tests/ballerina-spec-conformance-tests/build.gradle

```diff
@@ -46,9 +46,3 @@ test {
         result.exception?.printStackTrace()
     }
 }
-
-configurations {
-    testImplementation.exclude group: 'org.slf4j', module: 'slf4j-log4j12'
-    testImplementation.exclude group: 'org.codehaus.woodstox', module: 'woodstox-core-asl'
-    testImplementation.exclude group: 'org.codehaus.woodstox', module: 'stax2-api'
```

Comment:
Do we need to remove all of this?

Reply:
For the XML issue only we need to remove `'org.codehaus.woodstox', module: 'stax2-api'`. Is there any reason to exclude `group: 'org.codehaus.woodstox', module: 'woodstox-core-asl'` and `'org.slf4j', module: 'slf4j-log4j12'`

Reply:
If they are not required, we can exclude?

Reply:
Will update the PR.